analytic-ui-generator/src/main/java/com.wipro.ats.bdre.augen/AnalyticUIGeneratorMain.java

```diff
@@ -0,0 +1,110 @@
+package com.wipro.ats.bdre.augen;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This file "AnalyticUIGeneratorMain.java" should be located in "com/wipro/ats/bdre/augen" directory, not in "/var/lib/jenkins/jobs/BDRE-Github-Code-Quality/workspace/analytic-ui-generator/src/main/java/com.wipro.ats.bdre.augen". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1598)